File: Model/viewModels/SymptomGruppeListModel.cs
using System.ComponentModel.DataAnnotations;

namespace obligDiagnoseVerktøyy.Model.entities
{
    public class SymptomGruppeListModel
    {
        public int symptomGruppeId { get; set; }
        public string navn { get; set; } //Forklaringen kan hentes ut herfra

        public string beskrivelse { get; set; } //Forklaringen kan hentes ut herfra

    }
}

File: ClientApp/src/models/SymptomGruppeListModel.ts
export class SymptomGruppeListModel {
  symptomGruppeId: Number;
  navn: String;
  beskrivelse: String;

  doShow: boolean = false;

  constructor(navn: String, beskrivelse: String, symptomGruppeId: Number) {
    this.navn = navn;
    this.beskrivelse = beskrivelse;
    this.symptomGruppeId = symptomGruppeId;

  }

}
